core/tests/behavior/async_presign.rs

```diff
@@ -138,3 +139,9 @@ pub async fn test_presign_read(op: Operator) -> Result<()> {
     op.delete(&path).await.expect("delete must succeed");
     Ok(())
 }
+
+/// Presign delete should succeed.
+pub async fn test_presign_delete(op: Operator) -> Result<()> {
+    todo!();
```

Comment:
and how to write delete test here?

Reply:
Like we do in `test_presign_read`, we can create a file and send a `DELETE` request to the returning uri.